### ISSUE TITLE
In-app logs not updating appropriately

### ISSUE THREAD

**Issue by mchowning** [Automattic/pocket-casts-android#394]
### Description

The in-app logs that get sent to support seem to have started not updating correctly. In particular, it seems that the logs work fine up to a given point in time, but then there is a gap until the very last log line. It's almost as if instead of appending to the logs, new logs are instead just overwritten onto the last log line.

For example, I experienced an unrelated bug this morning, but when I sent the logs to myself, they looked like this:

```
I 04/10 13:28:20 Worker Downloading episode Jeremiah 90cf4c8b-aafe-4cb1-8ff9-5732053aad66
I 04/10 13:28:20 UpNextSyncJob - scheduled
I 11/10 08:54:45 stopForeground state: 2 (update notification)
```

Note that those logs show only a single entry since October 4th. It was inside of that gap that I encountered the bug I was trying to report, so I know I was using the phone during that time. If you're running a build of the app that never properly saved the logs, then it seems you will always only have a single line of logs.

This issue did not exist in the 7.22.0 release.

internal ref: p1665491925690749/1665479608.840769-slack-C02A333D8LQ

### Step-by-step reproduction instructions

1. Open the app
2. Do some things that would generate logs
3. Take a look at the log file on the phone (`data/data/au.com.shiftyjelly.pocketcasts.debug/files/logs/`)
4. Observe that there is a gap in the logs (or there is only a single log line)

### Did you search for existing bug reports?

- [X] I have searched for existing bug reports.

### Device, Operating system, and Pocket Casts app version

Pixel 6 Pro, Android 13, PC 7.23
Pixel 3, Android 12, PC main branch at https://github.com/Automattic/pocket-casts-android/commit/0765a874332d75c183d2f3c8bb45cbb61ab0b90c

**Comment by mchowning** [comment]
I believe this is due to some byte-code manipulation that Sentry is doing to the File IO classes. I should have a fix up shortly.

**Comment by mchowning** [comment]
This was fixed by #395
